### ISSUE TITLE
How does packet еncoding work?

### ISSUE THREAD

**Issue by MakTonda** [sntcz/LoRa.Message#4]
Could you please demonstrate an example of еncoding packets for LoRa/LoRaWAN? What methods to use, what parameters?
Тhank you!

**Comment by tkouba** [comment]
Only decoding is supported

**Comment by MakTonda** [comment]
Are you planning to support encoding?

**Comment by tkouba** [comment]
There is no plan to support encoding. But all contributions are welcome.

**Comment by alphapats** [comment]
Is there any tutorial on how to use this code?

**Comment by tkouba** [comment]
No, but there are tests and program to packet decode https://github.com/sntcz/LoRa.Message/blob/master/LoRaPacket/Program.cs
